zerver/lib/send_email.py

```diff
@@ -28,8 +32,15 @@ def build_email(template_prefix, to_email, from_email=None, reply_to_email=None,
     message = loader.render_to_string(template_prefix + '.txt',
                                       context=context, using='Jinja2_plaintext')
     html_message = loader.render_to_string(template_prefix + '.html', context)
-    if from_email is None:
-        from_email = settings.NOREPLY_EMAIL_ADDRESS
+
+    # We check for from_name == "" becacuse in forms.py we overwrite the Django send_mail function
```

Comment:
There are places where the `from_name` and `from_address` are set depending on the result of a conditional block (e.g. when we check for whether a specific welcome email sender has been specified, or for missed message emails where we check whether missed PM emails should be sent as a user), so in those cases we would want to check whether the input value is None or not.  We can get rid of the "" since we no longer use `parseaddr(VARIABLE)[0]` anywhere, though.